app/helpers/application_helper/toolbar/middleware_messaging_center.rb

```diff
@@ -0,0 +1,36 @@
+# noinspection RubyArgCount
+class ApplicationHelper::Toolbar::MiddlewareMessagingCenter < ApplicationHelper::Toolbar::Basic
+  button_group('middleware_messaging_monitoring', [
+    select(
+      :middleware_messaging_monitoring_choice,
+      'product product-monitoring fa-lg',
+      t = N_('Monitoring'),
```

Comment:
Well from what I understand we collect metrics for all resources, and JMS is just different type of resource. But if it's not needed I can remove this.